DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -340,6 +344,31 @@ public void ApproveDelegateUsers(params int[] ids)
             );
         }
 
+        public void RemoveDelegateUser(int delegateId)
+        {
+            using var transaction = new TransactionScope();

```

Comment:
Just double checking that the register stored procedure doesn't create Progress and aspProgress entry (Kevin seemed to think it might, but might have misremembered)?

Reply:
I didn't think so but I'll take another look, best to check carefully

Reply:
No hits for 'progress' in https://github.com/TechnologyEnhancedLearning/DLS-LEGACY/blob/master/database/dbo.uspSaveNewCandidate_V10.StoredProcedure.sql

Reply:
As discussed - no issues with this.